frontends/web/src/locales/de/app.json

```diff
@@ -1217,6 +996,75 @@
+      "title": "Backup-Hinweise"
+    },
+    "disable": "Passphrase deaktivieren",
+    "disableInfo": {
+      "button": "Deaktivieren",
+      "message": "Nachdem du die Passphrase deaktiviert hast, wirst du nach dem Entsperren deiner BitBox02 nicht mehr aufgefordert, eine Passphrase einzugeben. Daher wirst du deine Standard-Wallet aufrufen.\n\nAlle Coins in deiner Passphrase-Wallet befinden sich weiterhin in dieser Wallet, aber du kannst nicht darauf zugreifen, weil du nach dem Entsperren deiner BitBox02 deine Standard-Wallet öffnest.\n\nUm wieder auf deine Passphrase-Wallet zuzugreifen, aktiviere einfach wieder die Passphrase-Funktion und gib die entsprechende Passphrase ein, nachdem du die BitBox02 entsperrt hast.\n\n<strong>Tipp:</strong> Du kannst deine ursprüngliche Wallet immer noch öffnen, indem du die Passphrase leer lässt."
+    },
+    "enable": "Passphrase aktivieren",
+    "error": {
+      "e104": "Änderung der Passphrase-Einstellung wurde abgebrochen. "
+    },
+    "how": {
+      "button": "So sieht es aus",
+      "message": "Eine Passphrase funktioniert nicht wie ein Passwort, wie du es kennst. Wenn du deine Passphrase falsch eingibst, wirst du nicht benachrichtigt. Das liegt daran, dass <strong>jede Passphrase eine andere, aber gültige Wallet</strong> erstellt. Das heißt, du kannst mehrere Passphrasen für so viele Wallets verwenden, wie du willst. Auf jede Wallet kannst du aber nur zugreifen, wenn du die entsprechende Passphrase eingibst.\n\nWenn du deine BitBox02 einsteckst, wirst du wie üblich nach dem Gerätepasswort gefragt. Danach wirst du aufgefordert, eine Passphrase auf dem Gerät einzugeben.\n\nNachdem du die Passphrase eingegeben hast, wird dir die eingegebene Passphrase angezeigt. So kannst du bestätigen, dass du sie richtig eingegeben hast.",
+      "title": "Wie funktioniert die Passphrase?"
+    },
+    "intro": {
+      "message": "Eine Passphrase bietet zusätzliche Sicherheit für dein Wallet.\nHier erfährst du, wie sie funktioniert.",
+      "title": "Passphrase einrichten"
+    },
+    "progressDisable": {
+      "message": "Bestätige auf deiner BitBox, dass du die optionale Passphrase <strong>deaktivieren</strong> möchtest.",
+      "title": "Bestätige am Gerät"
+    },
+    "progressEnable": {
+      "message": "Bestätige auf deiner BitBox, dass du die optionale Passphrase <strong>aktivieren</strong> möchtest.",
+      "title": "Bestätige am Gerät"
+    },
+    "successDisabled": {
+      "message": "Optionale Passphrase <strong>erfolgreich aktiviert</strong>!\nVon nun an wirst du aufgefordert, eine Passphrase anzugeben.",
+      "messageEnd": "Bitte stecke die BitBox02 jetzt wieder ein.",
+      "title": "Passphrase aktiviert"
+    },
+    "successEnabled": {
+      "message": "Optionale Passphrase <strong>erfolgreich deaktiviert</strong>!\nVon nun an wirst du aufgefordert, eine Passphrase anzugeben.",
```

Comment:
This is wrong, when it is successfully deactivated it should say "Von nun an wirst du nicht mehr aufgefordert eine Passphrase anzugeben." or similar.

Reply:
cc @jstrnbrg